### DISCUSSION TITLE
Documentation is not in line with implementation

### DISCUSSION THREAD

**Discussion by luvarqpp** [qos-ch/logback#685] [General]
[Here](https://github.com/qos-ch/logback/blob/3ef4f12e85838c84630cab03232bde4ee72e3c2a/logback-classic/src/main/java/ch/qos/logback/classic/layout/TTLLLayout.java#L19) is stated, what is equivalent pattern for pattern logging, but this logger does also log key-value pairs. See kvp method call at line 66... Please make documentation in line with implementation.

**Comment by luvarqpp** [comment]
Perhaps correct value for pattern layout would involve https://logback.qos.ch/manual/layouts.html#kvp keyword. I have not searched if given key-value pairs are printed in exactly same manner through.

**Comment by ceki** [comment]
@luvarqpp  Thank you for the heads up. Issue has been fixed in commit 4e069d34d9422a43511a
